webviz_subsurface/plugins/_well_analysis/_plugin.py

```diff
@@ -76,18 +77,13 @@ def __init__(
     ) -> None:
         super().__init__()
 
-        # This is used to format the buttons in the well overview tab
-        WEBVIZ_ASSETS.add(
-            Path(webviz_subsurface.__file__).parent
-            / "_assets"
-            / "css"
-            / "inplace_volumes.css"
-        )
+        self.error_message = ""
 
         self._ensembles = ensembles
         self._theme = webviz_settings.theme
 
         if ensembles is None:
+            self.error_message = 'Incorrect argument, must provide "ensembles"'
             raise ValueError('Incorrect argument, must provide "ensembles"')
```

Comment:
This code pattern was new to me. In some way the `self.error_message` gives value even though we on the next line rasies an exception?

Reply:
No, this doesn't make sense. This error layout thing is something the summer student has taken from the WLF boilerplate repo. But everywhere else in `webviz-subsurface` the error messages is seen in the terminal so I think I'll just remove that from here.